pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/TriggersTest.java

```diff
@@ -162,4 +175,102 @@ public void actualTriggerCorrectScope() throws Exception {
         expect("actualTriggerCorrectScope")
                 .go();
     }
-}
+
+    @LocalData
+    @Test
+    public void doNotRestartEqualTriggers() throws Exception {
```

Comment:
Same feedback as for this test in the other PR.  We need tests showing the behavior works when there are more than one trigger and shows that if the triggers are reordered they still are detected as unchanged.